docs/src/app/components/pages/components/RaisedButton/ExampleComplex.jsx

```diff
@@ -31,17 +30,11 @@ const RaisedButtonExampleComplex = () => (
     <RaisedButton
       label="Label after"
       labelPosition="after"
-      secondary={true}
+      primary={true}
       icon={<ActionAndroid />}
       style={styles.button}
     />
     <RaisedButton
-      label="Disabled"
-      disabled={true}
-      style={styles.button}
-    />
-    <br />
```

Comment:
Yes, but that's a fudge to hide an issue with linkButton buttons. #2801. Better to fix the issue that hide the problem, IMHO.